test/Spago/Install.purs

```diff
@@ -104,31 +104,32 @@ spec = Spec.around withTempDir do
         )
       spago [ "install", "either" ] >>= shouldBeSuccess
 
-    -- TODO: this is broken at the moment
-    -- Spec.it "installs a package version by branch name with / in it" \{ spago, fixture } -> do
-    --   spago [ "init" ] >>= shouldBeSuccess
-    --   let
-    --     conf = Init.defaultConfig
-    --       (mkPackageName "ddd")
-    --       (Just $ unsafeFromRight $ Version.parse "0.0.1")
-    --       "Test.Main"
-    --   FS.writeYamlFile Config.configCodec "spago.yaml"
-    --     ( conf
-    --         { workspace = conf.workspace # map
-    --             ( _
-    --                 { extra_packages = Just $ Map.fromFoldable
-    --                     [ Tuple (mkPackageName "nonexistent-package") $ Config.ExtraRemotePackage $ Config.RemoteGitPackage
-    --                         { git: "https://github.com/spacchetti/purescript-metadata.git"
-    --                         , ref: "spago-test/branch-with-slash"
-    --                         , subdir: Nothing
-    --                         , dependencies: Just $ Dependencies $ Map.singleton (mkPackageName "prelude") Nothing
-    --                         }
-    --                     ]
-    --                 }
-    --             )
-    --         }
-    --     )
-    --   spago [ "install", "nonexistent-package" ] >>= shouldBeSuccessErr (fixture "installs-with-slash.txt")
+    Spec.it "installs a package version by branch name with / in it" \{ spago } -> do
+      spago [ "init" ] >>= shouldBeSuccess
+      let
+        conf = Init.defaultConfig
+          { name: mkPackageName "ddd"
+          , withWorkspace: true
+          , setVersion: Just $ unsafeFromRight $ Version.parse "0.0.1"
+          , testModuleName: "Test.Main"
+          }
+      FS.writeYamlFile Config.configCodec "spago.yaml"
+        ( conf
+            { workspace = conf.workspace # map
+                ( _
+                    { extra_packages = Just $ Map.fromFoldable
+                        [ Tuple (mkPackageName "nonexistent-package") $ Config.ExtraRemotePackage $ Config.RemoteGitPackage
+                            { git: "https://github.com/spacchetti/purescript-metadata.git"
+                            , ref: "spago-test/branch-with-slash"
+                            , subdir: Nothing
+                            , dependencies: Just $ Dependencies $ Map.singleton (mkPackageName "prelude") Nothing
+                            }
+                        ]
+                    }
+                )
+            }
+        )
+      spago [ "install", "nonexistent-package" ] >>= shouldBeSuccess
```

Comment:
I'm not sure what `installs-with-slash.txt` is supposed to say, but if this succeeds, is that good enough?

Reply:
I think that's fine, but we should also look that the folder in the `.spago` folder has the name that we expect it to have